config/ftbquests/quests/chapters/pneumaticcraft_repressurized.snbt

```diff
@@ -1736,5 +1737,29 @@
 				player_command: false
 			}]
 		}
+		{
+			title: "Thermal Lagging"
+			x: -12.0d
+			y: -10.0d
+			subtitle: "The Prison for Heat"
+			description: ["Thermal Laggings are insulating covers for machines to help with keeping heat. They greatly reduce a heat loss to the environment, and you can interact with any machine thought them!"]
```

Comment:
`greatly reduce a heat loss to the environment` -> `greatly reduce heat loss`

`and you can interact with any machine thought them` -> `and you can interact with machines through them`